synapse/handlers/federation.py

```diff
@@ -101,6 +102,10 @@
 
 logger = logging.getLogger(__name__)
 
+soft_failed_event_counter = Counter(
+    "synapse_handlers_federation_events_soft_failed", ""
```

Comment:
This metric name is based off of the following counters already in the codebase. It looks like there is already inconsistency in the plurality

 - [`synapse_handlers_appservice_events_processed`](https://github.com/matrix-org/synapse/blob/aec2cf1c9832eac46e1d94a39e75490b02320555/synapse/handlers/appservice.py#L44)
 - [`synapse_handler_presence_notified_presence`](https://github.com/matrix-org/synapse/blob/aec2cf1c9832eac46e1d94a39e75490b02320555/synapse/handlers/presence.py#L75)
  - [`synapse_storage_events_persisted_events`](https://github.com/matrix-org/synapse/blob/aec2cf1c9832eac46e1d94a39e75490b02320555/synapse/storage/databases/main/events.py#L58)
   - [`synapse_storage_events_state_delta_single_event`](https://github.com/matrix-org/synapse/blob/aec2cf1c9832eac46e1d94a39e75490b02320555/synapse/storage/persist_events.py#L51)